client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -233,11 +237,16 @@ class ImageEditorCanvas extends Component {
 			canvasX = -50 * widthRatio - 100 * leftRatio,
 			canvasY = -50 * heightRatio - 100 * topRatio;
 
+		const canvasOffsetTop = canvas.offsetTop;
+		const canvasOffsetLeft = canvas.offsetLeft;
+		const canvasOffsetWidth = canvas.offsetWidth;
+		const canvasOffsetHeight = canvas.offsetHeight;
```

Comment:
Note that we also have `getBoundingClientRect` available if we need more information, and that any node measurement will cause a reflow, so ideally we should limit how often `updateCanvasPosition` is called.